src/app/icd/ICDManager.cpp

```diff
@@ -154,6 +155,8 @@ void ICDManager::UpdateOperationState(OperationalState state)
             mOperationalState           = OperationalState::ActiveMode;
             uint32_t activeModeInterval = IcdManagementServer::GetInstance().GetActiveModeInterval();
             DeviceLayer::SystemLayer().StartTimer(System::Clock::Timeout(activeModeInterval), OnActiveModeDone, this);
+            DeviceLayer::SystemLayer().StartTimer(System::Clock::Timeout(activeModeInterval - ICD_ACTIVE_TIME_JITTER_MS),
```

Comment:
We need a check that `activeModeThreshold - ICD_ACTIVE_TIME_JITTER_MS` doesn't wrap around (in the negative)